### PR TITLE
Toast docs

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

File: packages/core/stories/toast/toast.stories.tsx [hunk 1]
```diff
@@ -36,31 +36,45 @@ DefaultInfo.args = {
...
-        <strong>File access</strong>
-      </Text>
-      <div>Viewers of this file can see comments and suggestions. </div>
-    </div>
-  ),
-};
+export const Error: ComponentStory<typeof Toast> = () => (
+  <Toast status="error">
+    <ToastContent>
+      <div>
```
Comment (libertymayc) [inline packages/core/stories/toast/toast.stories.tsx:42]: is this div needed?

File: packages/core/stories/toast/toast.stories.tsx [hunk 2]
```diff
@@ -36,31 +36,45 @@ DefaultInfo.args = {
...
-        <strong>System error</strong>
-      </Text>
-      <div>Connection timed out. Failed to retrieve data. </div>
-    </div>
-  ),
-};
+export const Warning: ComponentStory<typeof Toast> = () => (
+  <Toast status="warning">
+    <ToastContent>
+      <div>
```
Comment (libertymayc) [inline packages/core/stories/toast/toast.stories.tsx:63]: div needed?

File: packages/core/stories/toast/toast.doc.stories.mdx [hunk 3]
```diff
@@ -16,19 +16,47 @@ import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
 
 # Toast
 
+Toast displays a small pop-up notification that provides short feedback and disappears quickly. It is displayed in response to a user action or system event that relates to a workflow or application outside the user’s current focus.
+
 <Canvas>
   <Story id="core-toast--default-info" />
```
Comment (origami-z) [inline packages/core/stories/toast/toast.doc.stories.mdx:22]: Doesn't load

Comment (alycrys) [reply packages/core/stories/toast/toast.doc.stories.mdx]: thanks

**Review by libertymayc — DISMISSED** [review]

**Review by navkaur76 — COMMENTED** [review]
I would split 'Error and Warning' out to each sit under their own heading like Info and Success, and update the definition of each status to the following: 

Info - Display an informational status message
Success - Signify the successful completion of a system or user action 
Error - Signify a system error
Warning - Signify a system alert

(These definitions align to what we have in the status indication pattern)

**Comment by alycrys** [comment]
> I would split 'Error and Warning' out to each sit under their own heading like Info and Success, and update the definition of each status to the following:
> 
> Info - Display an informational status message Success - Signify the successful completion of a system or user action Error - Signify a system error Warning - Signify a system alert
> 
> (These definitions align to what we have in the status indication pattern)

Thanks @navkaur76 . I've added the updates

**Review by libertymayc — APPROVED** [review]

**Review by origami-z — APPROVED** [review]

**Pull request merged**